ra/ra.go

```diff
@@ -373,6 +373,7 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, reque
 				fmt.Sprintf("unable to get existing validations for regID: %d, identifier: %s",
 					regID, identifier.Value))
 			ra.log.Warning(string(outErr))
+			return authz, outErr
```

Comment:
Fixed in https://github.com/letsencrypt/boulder/pull/2286/commits/b62e9c3795ef8dab8cceaed82f98ee04da22eda8